src/autopas/containers/verletListsCellBased/verletListsCells/VerletListsCells.h

```diff
@@ -101,20 +101,23 @@ class VerletListsCells
       }
     }
 
-    typename verlet_internal::VerletListGeneratorFunctor f(_neighborLists, _cellMap, this->getCutoff());
+    typename verlet_internal::VerletListGeneratorFunctor f(_neighborLists, _cellMap,
+                                                           this->getCutoff() + this->getSkin());
 
     switch (_buildTraversal) {
       case c08: {
         if (useNewton3) {
           auto buildTraversal = C08Traversal<LinkedParticleCell, typename verlet_internal::VerletListGeneratorFunctor,
```

Comment:
This pattern occurs 5 times. We should move this to a template function, which uses the traversal type as template parameter.

Reply:
reduced it to two times, using static bool selector.
further code reduction using decltype